sim/midas/src/main/cc/Makefile

```diff
@@ -57,9 +57,16 @@ $(OUT_DIR)/dramsim2_ini: $(testchip_dir)/dramsim2_ini
 $(OUT_DIR)/$(DESIGN).chain:
 	$(if $(wildcard $(GEN_DIR)/$(DESIGN).chain),cp $(GEN_DIR)/$(DESIGN).chain $@,)
 
-override CXXFLAGS += -I$(midas_dir) -I$(util_dir)
+# dromajo setup
```

Comment:
Why can't this be in FireSim's makefrag?

Reply:
I didn't try to put it in FireSim's `makefrag`. I can move it there if that's preferred.

Reply:
`firesim/sim/src/main/makefrag/firesim/Makefrag` sounds like the defacto place to add all of these shenanigans.

Reply:
Yeah. It's specific to the firechip target. + I'm trying to kill that other makefile.